help/deploy/complete-setup/configure-auth.md

```diff
@@ -40,7 +45,7 @@ confirmation and forgot password emails.
 
 ## Oauth authentication
```

Comment:
the paragraph is outdated, and maybe should be split to 2/3 different paragraphs like in Settings.

Reply:
@ksumag did you consider what I suggested?